src/ResourceManager/Sql/Commands.Sql/help/Stop-AzureRmSqlDatabaseActivity.md

```diff
@@ -0,0 +1,181 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
+Module Name: AzureRM.Sql
+ms.assetid: B5C909D7-6087-463A-83BF-99DD196B9862
+online version: 
+schema: 2.0.0
+---
+
+# Stop-AzureRmSqlDatabaseActivity
+
+## SYNOPSIS
+Cancels the asynchronous updateslo operation on the database.
+
+## SYNTAX
+
+```
+Stop-AzureRmSqlDatabaseActivity [-ServerName] <String> [-ElasticPoolName <String>] -DatabaseName <String>
+ [-OperationId <Guid>] [-ResourceGroupName] <String> [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The **Stop-AzureRmSqlDatabaseActivity** cmdlet cancels the asynchronous updateslo operation on the database.
+
+## EXAMPLES
+
+### Example 1: Cancel the asynchronous updateslo operation on the database
+```
+PS C:\>Stop-AzureRmSqlDatabaseActivity -ResourceGroupName "ResourceGroup01" -ServerName "Server01" -DatabaseName "Database01" -OperationId af97005d-9243-4f8a-844e-402d1cc855f5
+OperationId     : af97005d-9243-4f8a-844e-402d1cc855f5
```

Comment:
@jiawang003 nit: can we add a blank line between the command and the output? I think this would make the full example easier to read

Reply:
fixed